File: 1323-maximum-69-number/1323-maximum-69-number.java
class Solution {
    public int maximum69Number (int num) {
       StringBuilder numSB = new StringBuilder();
        numSB.append(num);

        for (int i = 0; i < numSB.length(); ++i) {
            if (numSB.charAt(i) == '6') {
                numSB.setCharAt(i, '9');
                break;
            }
        }
        return Integer.parseInt(numSB.toString()); 
    }
}

File: 1323-maximum-69-number/1323-maximum-69-number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        string numString = to_string(num);
        for (auto &currChar : numString) {
            if (currChar == '6') {
                currChar = '9';
                break;
            }
        }
        return stoi(numString);
    }
};